docs/language/contracts.rst

```diff
@@ -119,6 +114,9 @@ For the creation of a contract, the data account must the **first** element in s
 .. include:: ../examples/solana/create_contract_with_metas.sol
   :code: solidity
 
+The ordering of the account in the ``AccountMeta`` array matters and must follow the
+:ref:`IDL disposal <account_management>`.
```

Comment:
IDL disposal? Am I missing something?

Reply:
I've changed to `IDL ordering`. What do you think?